### DISCUSSION TITLE
Are the testing docker images?

### DISCUSSION THREAD

**Discussion by gerroon** [navidrome/navidrome#2604] [Q&A] [answered]
Hi

The docker page is 404. Are there testing/nightly images? I am using `deluan/navidrome:latest` in my compose normally

**Comment by deluan** [comment, marked as answer]
> The docker page is 404

Which docker page? 

> Are there testing/nightly images? 

You can use the `deluan/navidrome:develop` tag if you want the bleeding edge version, built from `master` as commits are merged/pushed

**Reply by gerroon** [reply]
Thanks I will try it.

https://hub.docker.com/r/deluan/navidrome
![image](https://github.com/navidrome/navidrome/assets/8519469/0d3fae62-0a3e-42ef-8cbc-32aebc9a52bc)

**Reply by deluan** [reply]
Seems to be a glitch from Docker Hub:
<img width="1394" alt="Screenshot 2023-11-12 at 12 40 21 PM" src="https://github.com/navidrome/navidrome/assets/331353/c362c001-b7b4-4dfa-94ee-dc59062872a0">
